docs/08_getting_started.md

```diff
@@ -1,7 +1,110 @@
 # Getting Started
-This section provides you with the steps that are required to use the extractor.
+This section provides you with the steps that are required to use the extractor in your android app.
 
-### Implement a Downloader
+
+## Configure project.
+### Add dependencies to gradle:
+
+1. **project** `settings.gradle`:
+
+```gradle
+...
+buildscript {
+    repositories {
+        ...
+        maven { url 'https://jitpack.io' }
+    }
+    ...
+}
+```
+
+
+
+2. module/app build.gradle:
+
+```gradle
+// NewPipe Libraries
+implementation 'com.github.TeamNewPipe:NewPipeExtractor:0.21.12'
```

Comment:
NewPipe versions are prefixed with a 'v', so it should be `NewPipeExtractor:v0.21.12`

Reply:
I changed this ;)

Reply:
> Why are is the PR trying to merge into the branch feat/getting_started and not master?

You are right, I open new one to merge directly to `master` ;)